### PR TITLE
Project1 task3

### PR DESCRIPTION
**Pull request by SVBhuvanChandra**
Login Page, Admin page, Project 1 finished

### CODE REVIEW

**Comment by SVBhuvanChandra** [comment]
imported books.csv and checked the same in Adminer

**Comment by SVBhuvanChandra** [comment]
> how are you indicating if the registration is not successfull

Hi, Ma'am, below is the code snippet where I am notifying the user if registration is unsuccessful due to similar username

image

**Comment by SVBhuvanChandra** [comment]
> where did u set the session

Please find the below code snippets from application.py

I've created session here and using in "/auth"

image

image

File: project1/import.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+app1.config["SESSION_PERMANENT"] = False
+app1.config["SESSION_TYPE"] = "filesystem"
+Session(app1)
+
+app1.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app1.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app1.app_context().push()
+
+db1.init_app(app1)
+db1.create_all()
```
Comment (manasa133) [inline project1/import.py]: let all the tables be created at once, no need of separately writing again for creating books table.

Comment (SVBhuvanChandra) [reply project1/import.py]: > let all the tables be created at once, no need of separately writing again for creating books table.


Hi Ma'am,

Thanks for the review. Actually I feel better keeping a new py file for books. To make myself free from confusion, I discussed with my team and created a seperate file claaed books.py.
If you consider my reason is valid, please close the request. Kindly let me know for any further modification.

Regards

Comment (manasa133) [reply project1/import.py]: I mean tables can be created from application.py itself as you have the same code over there, get the instance of db in books.py to insert

Comment (SVBhuvanChandra) [reply project1/import.py]: Hi Ma'am, Thanks for the suggestion.
I have removed the retundat code and made a single file for creating databases. 
removed addiotnal python file an checked the code is working fine now.

**Comment by manasa133** [comment]
Merge with master branch

**Pull request merged**